code/hdb/hdbstandard.q

```diff
@@ -1,7 +1,26 @@
 // reload function
 reload:{
+	
+	if[.z.w in key .hdb.reloadcalls;
+		.hdb.reloadcalls[.z.w]:1b;
+		.lg.o[`reload;"reload call received from handle ", string[.z.w], "; reload calls pending from handles ", ", "sv string where not .hdb.reloadcalls];
```

Comment:
this log line is possibly confusing when there are no pending reload calls? maybe that last part should be conditional on there being pending calls?